diff --git a/packages/BUILD.bazel b/packages/BUILD.bazel
index f386a90..cb47778 100644
--- a/packages/BUILD.bazel
+++ b/packages/BUILD.bazel
@@ -1511,3 +1511,9 @@ filegroup(
     name = "was",
     srcs = ["@bazel_latex_texlive//:texlive_texmf__texmf-dist__tex__latex__was"],
 )
+
+latex_package(
+    name = "seqsplit",
+    srcs = ["@bazel_latex_texlive//:texlive_texmf__texmf-dist__tex__latex__seqsplit"],
+    tests = ["seqsplit_test.tex"],
+)
diff --git a/packages/seqsplit_test.tex b/packages/seqsplit_test.tex
new file mode 100644
index 0000000..802bc47
--- /dev/null
+++ b/packages/seqsplit_test.tex
@@ -0,0 +1,5 @@
+\documentclass{article}
+\usepackage{seqsplit}
+\begin{document}
+\seqsplit{"aaaaaaa"}  % produces `a.txt`
+\end{document}
